internal/store/migrations.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package store
+
+import (
+	"github.com/blang/semver"
+)
+
+type migration struct {
+	fromVersion   semver.Version
+	toVersion     semver.Version
+	migrationFunc func(execer) error
+}
+
+// migrations defines the set of migrations necessary to advance the database to the latest
+// expected version.
+//
+// Note that the canonical schema is currently obtained by applying all migrations to an empty
+// database.
+var migrations = []migration{
+	{semver.MustParse("0.0.0"), semver.MustParse("0.1.0"), func(e execer) error {
```

Comment:
Small request here. Considering this is the initial commit of genesis, we only really need one batch of migrations. I see there is a separate migration below, which totally works, but we could combine them to simplify things a bit.

Reply:
Yeah totally agree. I was planning to do this and then forgot. Thanks Gabe ❤️